### PR TITLE
Add Inorder Traversal

### PR DESCRIPTION
**Pull request by muthuannamalai12**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Added Inorder Traversal in C 
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1923 
(Write your answer here.)

### CODE REVIEW

**Comment by muthuannamalai12** [comment]
@srimani-programmer . Kindly Review

**Comment by muthuannamalai12** [comment]
Kindly review it @atarax665

File: C/ds/Inorder.c [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		tree->right = insert(tree->right, digit);
+	return tree;
+}
+
+void inorder(node *t)
+{
+	if (t != NULL)
+	{
+		inorder(t->left);
+		printf(" %d", t->data);
```
Comment (atarax665) [inline C/ds/Inorder.c]:
```suggestion
		printf("%d ", t->data);
```

File: C/ds/Inorder.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Inorder Traversal
```
Comment (atarax665) [inline C/ds/Inorder.c]: Use multi-line comment instead of multiple single line comments.

File: C/ds/Inorder.c [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+Sample Output
+
+Enter integer:To quit enter 0                                                                                                   
+12 4 6 9 14 17 3 19 0                                                                                                           
+                                                                                                                                
+The inorder traversal of tree is:                                                                                               
+19 17 9 4 12 6 14 3 
+
+*/
+
```
Comment (atarax665) [inline C/ds/Inorder.c]: Remove these redundant blank lines.

File: C/ds/Inorder.c [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	printf("\nThe inorder traversal of tree is:\n");
+	inorder(root);
+	getchar();
+}
+
+
+/*
+
+Sample Output
```
Comment (atarax665) [inline C/ds/Inorder.c]: Add Space and Time complexity of the program too.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have changed it. Kindly review it @atarax665

**Review by atarax665 — APPROVED** [review]

**Comment by muthuannamalai12** [comment]
@iamrajiv Please Review

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**